osutil/mountentry_linux.go

```diff
@@ -235,224 +140,3 @@ func MountOptsToFlags(opts []string) (flags int, err error) {
 	}
 	return flags, nil
 }
-
-// OptStr returns the value part of a key=value mount option.
```

Comment:
@chipaca I think this part could be broken out and landed separately as this area is moving and it will conflict otherwise. The rest should be much more chewable then.

Reply:
noted